packages/material-ui/src/SpeedDialIcon/SpeedDialIcon.test.js

```diff
@@ -22,57 +23,50 @@ describe('<SpeedDialIcon />', () => {
   }));
 
   it('should render the Add icon by default', () => {
-    const wrapper = mount(<SpeedDialIcon />);
-    expect(findOutermostIntrinsic(wrapper).find('svg[data-testid="AddIcon"]').length).to.equal(1);
+    const { getAllByTestId } = render(<SpeedDialIcon />);
+    expect(getAllByTestId('AddIcon').length).to.equal(1);
   });
 
   it('should render an Icon', () => {
-    const wrapper = mount(<SpeedDialIcon icon={icon} />);
-    const iconWrapper = findOutermostIntrinsic(wrapper).childAt(0);
-    expect(iconWrapper.find(Icon).length).to.equal(1);
+    const { container } = render(<SpeedDialIcon icon={icon} />);
+    expect(container.firstChild).to.have.class(classes.root);
```

Comment:
This doesn't test for the presence of an icon.

Reply:
Updated